src/numpy-stubs/__init__.pyi

```diff
@@ -27,6 +27,8 @@ from typing import (
 )
 from typing_extensions import Buffer, CapsuleType, LiteralString, Never, Protocol, Self, TypeVar, Unpack, deprecated, override
 
+import numpy as np
```

Comment:
There's no need for this (and in some cases it leads to weird bugs in mypy or pyright because of the cyclical references)